test/ModalSpec.js

```diff
@@ -85,4 +85,16 @@ describe('Modal', function () {
     assert.ok(dialog.className.match(/\bmodal-sm\b/));
   });
 
+  it('Should pass dialogClassName to the dialog', function () {
+    let noOp = function () {};
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Modal dialogClassName="testCss" onRequestHide={noOp}>
+        <strong>Message</strong>
+      </Modal>
+    );
+
+    let dialog = instance.getDOMNode().getElementsByClassName('modal-dialog')[0];
+    assert.ok(dialog.className.match(/\btestCss\b/));
```

Comment:
`React.findDOMNode(instance)` should be used in place of `component.getDOMNode()`. You could also use array destructuring to grab the first element rather than reference directly by index:
```javascript
let [dialog] = node.getElementsByClassName('modal-dialog');
```

You can use `should` or `expect` in place of this `assert.ok` to chain together a more readable assertion:
```javascript
dialog.className.should.match(/\btestCss\b/); //with should
expect(dialog.className).to.match(/\btestCss\b/); //with expect
assert.match(dialog.className, /\btestCss\b/); //with assert
```